src/libraries/Common/src/System/Net/Http/aspnetcore/Http2/Hpack/Huffman.cs

```diff
@@ -11,7 +11,7 @@ internal static class Huffman
     {
         // HPack static huffman code. see: https://httpwg.org/specs/rfc7541.html#huffman.code
         // Stored into two tables to optimize its initialization and memory consumption.
-        private static readonly uint[] s_encodingTableCodes = new uint[257]
+        private static ReadOnlySpan<uint> EncodingTableCodes => new uint[257]
```

Comment:
Do we have a proposal for an analyzer that can suggest such changes?

Reply:
Yup: https://github.com/dotnet/runtime/issues/33780